### DISCUSSION TITLE
[RFC] Merge Leaf Router into Core

### DISCUSSION THREAD

**Discussion by mychidarko** [leafsphp/leaf#250] [Polls]
We're planning to merge the router package back into Leaf's core, as the benefits of keeping it separate are no longer relevant. Leaf's core now offers only the essential, low-level features, so the incremental addition of Leaf is no longer a concern.

By merging the router, you'll experience substantial performance improvements, all without any changes to the existing API.

**Comment by terrybr** [comment]
If that can increase performance and there are no benefits of keeping it separate, I'm all for merging Leaf Router into Core.

**Comment by mychidarko** [comment]
We have pretty much gone ahead with this change. We hope to publish the update during the next couple of days after extensive testing
